apps/docs/src/pages/components/radio-group.mdx

```diff
@@ -0,0 +1,130 @@
+import {
+  HvRadio,
+  HvRadioGroup,
+  radioGroupClasses,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+
+import Playground from "@docs/components/code/Playground";
+import { Description } from "@docs/components/page/Description";
+import { Page } from "@docs/components/page/Page";
+import { getComponentData } from "@docs/utils/component";
+
+export const getStaticProps = async ({ params }) => {
+  const meta = await getComponentData("RadioGroup", "core", radioGroupClasses);
+  return { props: { ssg: { meta: meta } } };
+};
+
+<Description />
+
+<Page>
+
+<Playground
+  Component={HvRadioGroup}
+  componentName="HvRadioGroup"
+  controls={{
+    label: {
+      type: "text",
+      defaultValue: "Visualization type",
+    },
+    description: {
+      type: "text",
+      defaultValue: "Select how you want to visualize your dataset",
+    },
+    orientation: {
+      defaultValue: "vertical",
+    },
+    required: {
+      defaultValue: true,
+    },
+    disabled: {
+      defaultValue: false,
+    },
+  }}
+  children={
+    <>
+      <HvRadio label="Bar Chart" value="1" />
+      <HvRadio label="Line Chart" value="2" />
+      <HvRadio label="Pie Chart" value="3" />
+    </>
+  }
+/>
```

Comment:
we should be able to pass `children` normally (without needing the `Fragment`), but there's something wrong with it

ie.
```suggestion
>
  <HvRadio label="Bar Chart" value="1" />
  <HvRadio label="Line Chart" value="2" />
  <HvRadio label="Pie Chart" value="3" />
</Playground>
```

Reply:
https://github.com/lumada-design/hv-uikit-react/pull/4469

Reply:
I'll wait for that to be merged and update this.

Reply:
I'll wait for your review there! 😆

Reply:
Updated.